### ISSUE TITLE
Support custom renaming logic

### ISSUE THREAD

**Issue by nex3** [google/postcss-rename#15]
Currently, the class renamer only supports its own built-in renaming logic. It should also support passing in a custom `string -> string` function that can rewrite names based on user-provided logic.

**Comment by nex3** [comment]
I think a TS-y way of supporting this would be to make `Options.type` take a union type of strings (representing built-in renamers) and a function. For example:

```ts
class Options {
  // ...
  type: "none" | "debug" | "minimal" | (name: string) -> string;
}
```

**Comment by nex3** [comment]
Fixed by #22.
